server/sequence.js

```diff
@@ -32,11 +28,11 @@ let MySequence = class MySequence {
 };
 
 MySequence = decorate([
-  param(0, context.inject(SequenceActions.FIND_ROUTE)),
-  param(1, context.inject(SequenceActions.PARSE_PARAMS)),
-  param(2, context.inject(SequenceActions.INVOKE_METHOD)),
-  param(3, context.inject(SequenceActions.SEND)),
-  param(4, context.inject(SequenceActions.REJECT)),
+  function (target, key) { context.inject(SequenceActions.FIND_ROUTE)(target, key, 0); },
```

Comment:
This does not seem like an improvement to me - the parameter index is still hard-coded.